puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/puppet-deploy.yaml

```diff
@@ -0,0 +1,13 @@
+- job:
+    name: puppet-deploy
+    project-type: workflow
+    properties:
+      - github:
+          url: https://github.com/theforeman/foreman-infra
+    triggers:
+      - github
```

Comment:
But is also triggered by git hooks from GitHub? Why do we need both?

Reply:
See https://wiki.jenkins.io/display/JENKINS/GitHub+Plugin -- the two are inter-related